src/Components/Articles.css

```diff
@@ -0,0 +1,62 @@
+.center {
+  border: 5px solid #FFFF00;
+  text-align: right;
+}
+.content {
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.newsCard {
+  display: flex;
+  width: 150%;
+  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.16), 0 2px 10px 0 rgba(0, 0, 0, 0.12);
+  padding: 10px;
+  margin-bottom: 25px;
+  border-radius: 5px;
+}
+
+.newsText {
+  padding: 0 0 0 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+.newsImage {
+  object-fit: cover;
+  width: 320px;
+  height: 268px;
+}
+
+.category{
+  font-size: 15px;
+  line-height: 27px;
+  font-family: "Roboto", sans-serif;
+}
+.title {
+  font-size: 23px;
+  line-height: 27px;
+  font-family: "Roboto", sans-serif;
+}
+
+
+
+.description {
+  font-size: 16px;
+  line-height: 22px;
+  color: #44444d;
+}
+
+@media (max-width: 800px) {
+  .newsCard {
+    flex-direction: column;
+    width: 100%;
+  }
+
+  .newsImage {
+    width: 100%;
+    height: 100%;
+    padding-bottom: 20px;
+  }
+}
```

Comment:
I don't love that there is a css file in the Components directory. Consider moving it to a style directory.